src/Header/ContentLength.php

```diff
@@ -38,12 +38,21 @@ public static function fromString($headerLine)
         return $header;
     }
 
-    public function __construct($value = null)
+    public function __construct($value)
     {
-        if ($value) {
-            HeaderValue::assertValid($value);
-            $this->value = $value;
+        $this->setFieldValue($value);
+    }
+
+    public function setFieldValue($value)
```

Comment:
Adding a `setFieldValue()` function is considered a new feature and it should be done with a new PR.